config/v1/types_scheduling.go

```diff
@@ -93,6 +89,22 @@ var (
 	NoScoring SchedulerProfile = "NoScoring"
 )
 
+// ProfileCustomizations contains various parameters for modifying the default behavior of certain profiles
+type ProfileCustomizations struct {
+	// experimentalDynamicResourceAllocation enables/disables dynamic resource allocation feature.
+	// +kubebuilder:default=Disabled
+	ExperimentalDynamicResourceAllocation DRAEnablement `json:"experimentalDynamicResourceAllocation"`
```

Comment:
We shouldn't use `experimental`, but instead use the field name we intend to have in the product, and mark it as a featuregated field instead

Reply:
How does one generates the TPNU and CustomNU CRDs? I searched through the o/api repo though did not find any script. Are we expected to create these by hand?

Reply:
There are a couple of things to do, mostly documented in that doc you found. You can follow pretty much everything there, but not the Makefile changes, they won't be needed.

The upshot is to copy/paste the existing CRDs, correct the names, correct the feature set annotations, and run the generators. Your fields will need to be marked as featureset TPNU and CNU, which is exampled in that doc

Reply:
How do I run the generators? Is there any example?

Reply:
`make update` or, try the various scripts in `./hack` to run them separately. For crd schemas it's `./hack/update-codegen-crds.sh`, you'll want to run all of the generators at some points though

Reply:
Done